TOCSharp/TOCClient.cs

```diff
@@ -195,6 +200,18 @@ private async Task SendCommandAsync(params string[] args)
             });
         }
 
+        private async Task SendKeepAliveAsync()
```

Comment:
KeepAlive methods should be moved to the FLAP connection class as that's a connection thing

Reply:
The SendKeepAliveAsync task has been moved. If the actual loop itself needs to be moved too, let me know.